src/plugins/syntax-highlighting/bracket-pair-colorization.ts

```diff
@@ -0,0 +1,47 @@
+export function generateBracketPairColorizationCSS(
+  colors: [number, number, number][],
+  colorInText: boolean,
+  thickenBrackets: number
+) {
+  if (!colors[0]) return [];
+
+  const colorMaker = `rgb(${colors[0]
+    .map((_, i) => {
+      return `calc(${colors
+        .map((col, colindex) => {
+          const channel = col[i];
+          return `${channel} * max(0, 10000 * cos(
+            2 * 3.14159265358979323 * (var(--test2) - ${colindex}) / ${colors.length}
+            ) - 9999)`;
+        })
+        .join(" + ")})`;
+    })
+    .join(", ")})`;
+
+  return [
+    `
+    .dcg-mq-root-block {
+        --test1: 0;
+        --test2: 0;
```

Comment:
Do these have more descriptive names than `test`?

Reply:
Yeah, those are placeholders that I forgot to change lol

Reply:
fixed